### PR TITLE
[Docs] add faq to `How to install mmdeploy on Jetsons`

### PR DESCRIPTION
**Pull request by AllentDan**
Add a FAQ to `How to install mmdeploy on Jetsons`

### CODE REVIEW

File: docs/en/tutorials/how_to_install_mmdeploy_on_jetsons.md [hunk 1]
```diff
@@ -117,3 +117,11 @@ Then install onnx through:
 pip install onnx
 ```
 Then reinstall mmdeploy.
+
+
+### FAQs
+
+- For TX2, `#assertion/root/workspace/mmdeploy/csrc/backend_ops/tensorrt/batched_nms/trt_batched_nms.cpp,98 2022-02-08 07:15:09,334 - mmdeploy - ERROR - visualize tensorrt model failed.`
```
Comment (grimoire) [inline docs/en/tutorials/how_to_install_mmdeploy_on_jetsons.md:124]: Not just tx2, all devices with arch==7.2 have the same problem.

File: docs/en/tutorials/how_to_install_mmdeploy_on_jetsons.md [hunk 2]
```diff
@@ -117,3 +117,11 @@ Then install onnx through:
 pip install onnx
 ```
 Then reinstall mmdeploy.
+
+
+### FAQs
+
+- For devices with arch==7.2 , `#assertion/root/workspace/mmdeploy/csrc/backend_ops/tensorrt/batched_nms/trt_batched_nms.cpp,98 2022-02-08 07:15:09,334 - mmdeploy - ERROR - visualize tensorrt model failed.`
```
Comment (grimoire) [inline docs/en/tutorials/how_to_install_mmdeploy_on_jetsons.md:124]: the user might not understand what is arch==7.2, use jetson tx2, and jetson nano instead.
And we can add an assert in the nms plugin if arch==7.2.

File: csrc/backend_ops/tensorrt/batched_nms/allClassNMS.cu [hunk 3]
```diff
@@ -85,6 +87,10 @@ __global__ void allClassNMS_kernel(const int num, const int num_classes,
   //__shared__ bool kept_bboxinfo_flag[CAFFE_CUDA_NUM_THREADS * TSIZE];
   extern __shared__ bool kept_bboxinfo_flag[];
 
+#if defined(__CUDA_ARCH__) && __CUDA_ARCH__ <=720
```
Comment (grimoire) [inline csrc/backend_ops/tensorrt/batched_nms/allClassNMS.cu:90]: can we move assert out of the kernel?

Comment (AllentDan) [reply csrc/backend_ops/tensorrt/batched_nms/allClassNMS.cu:90]: Seems we can't from [this blog](https://www.rookiehpc.com/cuda/docs/__cuda_arch__.php). While in this [blog](https://llvm.org/docs/CompileCudaWithLLVM.html), `__CUDA_ARCH__` will be defined if NVCC is using clang as a preprocessor. Maybe we can do the check for some threads like the first thread?

Comment (grimoire) [reply csrc/backend_ops/tensorrt/batched_nms/allClassNMS.cu:90]: @lvhan028 any idea?

Comment (lvhan028) [reply csrc/backend_ops/tensorrt/batched_nms/allClassNMS.cu:90]: I think we can try `static_assert`, which happens at compiling time

Comment (lvhan028) [reply csrc/backend_ops/tensorrt/batched_nms/allClassNMS.cu:90]: I am not sure whether it brings in `Undefined Behavior`

Comment (lzhangzz) [reply csrc/backend_ops/tensorrt/batched_nms/allClassNMS.cu:90]: The documentation for `__CUDA_ARCH__` says that host code should not depend on it. However, we can query the compute capability of the device using `cudaDeviceGetAttribute`

File: csrc/backend_ops/tensorrt/batched_nms/allClassNMS.cu [hunk 4]
```diff
@@ -251,6 +250,11 @@ pluginStatus_t allClassNMS(cudaStream_t stream, const int num, const int num_cla
                            const bool isNormalized, const DataType DT_SCORE, const DataType DT_BBOX,
                            void *bbox_data, void *beforeNMS_scores, void *beforeNMS_index_array,
                            void *afterNMS_scores, void *afterNMS_index_array, bool flipXY) {
+  auto __cuda_arch__ = get_cuda_arch(0);  // assume there is only one arch 7.2 device
+  if (__cuda_arch__ == 720 && top_k > 200) {
```
Comment (grimoire) [inline csrc/backend_ops/tensorrt/batched_nms/allClassNMS.cu:254]: Why 200?

Comment (AllentDan) [reply csrc/backend_ops/tensorrt/batched_nms/allClassNMS.cu:254]: Just a number from experience.

Comment (grimoire) [reply csrc/backend_ops/tensorrt/batched_nms/allClassNMS.cu:254]: Too small for object detection task.

**Review by grimoire — APPROVED** [review]

**Comment by grimoire** [comment]
Errr, I am not sure if adding an assert is a good idea if we do not known the max topk.
Maybe a warning is enough?

**Review by grimoire — APPROVED** [review]
LGTM

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**